packages/client/ui/react-ui/src/providers/CrossmintWalletProvider.tsx

```diff
@@ -35,17 +57,24 @@ export type WalletConfig = WalletParams & { type: "evm-smart-wallet" };
 export function CrossmintWalletProvider({
     children,
     defaultChain,
+    showWalletModals = true,
```

Comment:
is there a valid scenario where we _wouldn't_ show these modals? 
Is it worth having this be toggleable or can we just assume always on and remove this?